test/contract/src/main.rs

```diff
@@ -194,7 +243,10 @@ pub fn main() -> i8 {
     test_partial_load_tx_hash();
     test_high_level_apis();
     test_query();
-    test_dynamic_loading();
+    // don't run test_dynamic_loading_c_impl at same time, 
+    // the stack is marked as executable already and can't be re-used
+    // test_dynamic_loading();
+    test_dynamic_loading_c_impl();
```

Comment:
Pls just take the CKBDLContext::new out of this function, so we are able to call it multiple times.

Reply:
> ```
> cargo publish --dry-run
> ```
> 
> returns an error
> 
> ```
> error: could not find native static library `dl-c-impl`, perhaps an -L flag is missing?
> 
> error: aborting due to previous error; 12 warnings emitted
> 
> error: could not compile `ckb-std`
> ```

Already fixed. the make target "build-dl-c-impl" or "build-dl-c-imp_via-docker" should be run before publish.
It's suggested to run test/integration/build-dl-c-impl before publishing.